applications/webhooks/src/webhooks_util.erl

```diff
@@ -181,11 +181,12 @@ maybe_fire_foldl(Key, Value, {_ShouldFire, JObj}) ->
 fire_hook(JObj, #webhook{custom_data = 'undefined'
                         } = Hook) ->
     EventId = kz_binary:rand_hex(5),
-    do_fire(Hook, EventId, JObj);
+    NewJObj = kz_json:insert_value(<<"cluster_id">>, kzd_cluster:id(), JObj),
+    do_fire(Hook, EventId, NewJObj);
 fire_hook(JObj, #webhook{custom_data = CustomData
                         } = Hook) ->
-    EventId = kz_binary:rand_hex(5),
-    do_fire(Hook, EventId, kz_json:merge_jobjs(CustomData, JObj)).
+    NewJObj = kz_json:merge_jobjs(CustomData, JObj),
+    fire_hook(NewJObj, Hook#webhook{custom_data = 'undefined'}).
```

Comment:
why this change ?

Reply:
nm

Reply:
To make it all calls go through one point so that there aren't repeated calls for adding stuff to the request

Reply:
yep, noticed that after, `nm` => `never mind`